### PR TITLE
Add paragraph about serverside rendering to README

### PR DESCRIPTION
**Pull request by schlessera**
## Summary

Adds a short paragraph with marketing buzz about the serverside rendering and the AMP optimizer.

## Checklist

- [ ] My pull request is addressing an open issue (please create one otherwise).
- [ ] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [ ] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

**Review by westonruter — APPROVED** [review]

**Review by amedina — APPROVED** [review]
LGTM.

File: readme.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Features and capabilities provided by the plugin include:
 - **Core Theme Support**: enabling AMP compatibility for all core themes, from Twenty Ten all the way through Twenty Twenty.
 - **Compatibility Tool**: when automatic conversion of markup to AMP is not possible, debug AMP validation errors with detailed information including the invalid markup and the specific components responsible on site (e.g theme, plugin, embed); validation errors are shown contextually with their respective blocks in the editor.
 - **CSS Tree Shaking**: automatically remove the majority of unused CSS to bring the total under AMP's 75KB limit; when the total after tree shaking is still over this limit, prioritization is used so that the all-important theme stylesheet important is retained, leaving less important ones to be excluded (e.g. print styles).
+- **Server-side Rendering**: full integration of the [AMP Optimizer library](https://github.com/ampproject/amp-toolbox/tree/master/packages/optimizer) providing automated server-side rendering of AMP pages, drastically improving the [Last Contenful Paint](https://web.dev/lcp/) metric.
```
Comment (westonruter) [inline readme.md]:
```suggestion
- **Server-side Rendering**: integrated PHP-port of the [AMP Optimizer](https://github.com/ampproject/amp-toolbox/tree/master/packages/optimizer) providing automated [server-side rendering](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/server-side-rendering/) of AMP pages, drastically improving the [Last Contentful Paint](https://web.dev/lcp/) (LCP) metric.
```

File: readme.txt [hunk 2]
```diff
@@ -20,6 +20,7 @@ Features and capabilities provided by the plugin include:
 - **Core Theme Support**: enabling AMP compatibility for all core themes, from Twenty Ten all the way through Twenty Twenty.
 - **Compatibility Tool**: when automatic conversion of markup to AMP is not possible, debug AMP validation errors with detailed information including the invalid markup and the specific components responsible on site (e.g theme, plugin, embed); validation errors are shown contextually with their respective blocks in the editor.
 - **CSS Tree Shaking**: automatically remove the majority of unused CSS to bring the total under AMP's 75KB limit; when the total after tree shaking is still over this limit, prioritization is used so that the all-important theme stylesheet important is retained, leaving less important ones to be excluded (e.g. print styles).
+- **Server-side Rendering**: full integration of the [AMP Optimizer library](https://github.com/ampproject/amp-toolbox/tree/master/packages/optimizer) providing automated server-side rendering of AMP pages, drastically improving the [Last Contenful Paint](https://web.dev/lcp/) metric.
```
Comment (westonruter) [inline readme.txt]:
```suggestion
- **Server-side Rendering**: integrated PHP-port of the [AMP Optimizer](https://github.com/ampproject/amp-toolbox/tree/master/packages/optimizer) providing automated [server-side rendering](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/server-side-rendering/) of AMP pages, drastically improving the [Last Contentful Paint](https://web.dev/lcp/) (LCP) metric.
```

**Review by westonruter — CHANGES_REQUESTED** [review]
I don't think we can say “full integration” of the AMP Optimizer because it's not actually integrating it. We have ported it. And it's not a full port.

File: readme.md (hunk 1 above)
Comment (westonruter) [reply readme.md]: Note I've also opened a PR to add a mention of the AMP plugin in the AMP docs for SSR: https://github.com/ampproject/amp.dev/pull/4076

**Comment by amedina** [comment]
> I don't think we can say “full integration” of the AMP Optimizer because it's not actually integrating it. We have ported it. And it's not a full port.

Great point/catch. I am going to make a small tweak...

**Comment by westonruter** [comment]
Suggestion of a suggestion of a suggestion:

image

I love it. Diff inception.

**Review by westonruter — APPROVED** [review]

**Review by westonruter — APPROVED** [review]

**Comment by schlessera** [comment]
The "full integration" was originally meant to refer to the AMP Optimizer library in the PHP ecosystem, for which we don't have a link yet, but which will soon be a thing.

**Comment by amedina** [comment]
Got it; that makes sense. For now the wording reads well and corresponds
precisely to the current state. We will calibrate as the library and the
integration evolve. The important point is to convey the fact that the AMP
plugin provides the capability of generating optimized AMP with server-side
rendering out of the box.

**Review by dnobles003 — COMMENTED** [review]
Thank you.

**Pull request merged**